apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -47,11 +47,17 @@ function tableName(table: string, schema?: string): string{
 /**
  * Do not convert DATE types to JS date.
  * It ignores of applying a wrong timezone to the date.
+ *
+ * See also: https://github.com/brianc/node-postgres/issues/285
+ * (and note that the code refrenced in /lib/textParsers.js has been broken out into it own module
+ * so it now lives in https://github.com/brianc/node-pg-types/blob/master/lib/textParsers.js#L175)
+ *
  * TODO: do not convert as well these same types with array (types 1115, 1182, 1185)
  */
 pg.types.setTypeParser(1082, 'text', (val) => val); // date
 pg.types.setTypeParser(1114, 'text', (val) => val); // timestamp without timezone
 pg.types.setTypeParser(1184, 'text', (val) => val); // timestamp
+pg.types.setTypeParser(1186, 'text', (val) => val); // interval (Issue #1442 "BUG: INTERVAL columns receive wrong value when cloning row)
```

Comment:
The `pg.types` object provides a `builtins` object that has constants for these numbers which might be easier to use:

```
pg.types.setTypeParser(pg.types.builtins.DATE, (val) => val);
pg.types.setTypeParser(pg.types.builtins.TIMESTAMP, (val) => val);
pg.types.setTypeParser(pg.types.builtins.TIMESTAMPTZ, (val) => val);
pg.types.setTypeParser(pg.types.builtins.INTERVAL, (val) => val);
```

Reply:
Thanks, @MasterOdin! Definitely will do that